src/services/refactors/convertLetToConst.ts

```diff
@@ -0,0 +1,157 @@
+/* @internal */
+namespace ts.refactor.generateGetAccessorAndSetAccessor {

```

Comment:
Should that really be called "generateGetAccessorAndSetAccessor"?

Reply:
Emmm.. Copy & Paste. You know.... :XD